lib/beautifier.js

```diff
@@ -201,15 +202,116 @@ class Beautifier {
                 n: 'trades'
             }
         };
+
+        this._floatKeys = [
+            'accumulatedQuantity',
+            'asks',
+            'askPrice',
+            'askQty',
+            'availableBalance',
+            'balances',
+            'baseAssetVolume',
+            'bestAskPrice',
+            'bestAskQuantity',
+            'bids',
+            'bestBid',
+            'bestBidQuantity',
+            'bidPrice',
+            'bidQty',
+            'buyerCommission',
+            'close',
+            'closeQuantity',
+            'closeTime',
+            'commission',
+            'currentClose',
+            'executedQty',
+            'free',
+            'high',
+            'highPrice',
+            'lastPrice',
+            'lowPrice',
+            'ignored',
+            'icebergQuantity',
+            'icebergQty',
+            'lastTradeQuantity',
+            'locked',
+            'low',
+            'makerCommission',
+            'onOrderBalance',
+            'open',
+            'openPrice',
+            'origQty',
+            'previousClose',
+            'prevClosePrice',
+            'price',
+            'priceChange',
+            'priceChangePercent',
+            'quantity',
+            'qty',
+            'quoteAssetVolume',
+            'quoteVolume',
+            'sellerCommission',
+            'stopPrice',
+            'takerBaseAssetVolume',
+            'takerCommission',
+            'takerQuoteAssetVolume',
+            'volume',
+            'weightedAveragePrice',
+            'weightedAvgPrice'
+        ];
+        // Map so we don't have to perform indexOf for each iteration
+        this._floatKeysHashMap = {};
+        _.each(this._floatKeys, function (keyName) {
+            inst._floatKeysHashMap[keyName] = 1;
+        });
+    }
+
+    beautifyKeyVal(key, val) {
+        if (this._floatKeysHashMap[key]) {
+            return parseFloat(val);
+        }
+        return val;
+    }
+
+    beautifyObject(data) {
+        var inst = this;
+        _.mapObject(data, function(val, key) {
+            if (_.isArray(val)) {
+              data[key] = inst.beautifyArray(val, key);
+            } else {
+              data[key] = inst.beautifyKeyVal(key, val);
+            }
+        });
+        return data;
+    }
+
+    beautifyArray(data, parentKey) {
+        var inst = this;
+        var result = [];
+        _.each(data, function (val, key) {
+          if (_.isArray(val)) {
```

Comment:
Also `Array.isArray(val)`. Prefer to use native methods where possible instead of increasing lodash dependency.